lib/SearchAndSort/SearchAndSort.js

```diff
@@ -369,7 +369,8 @@ class SearchAndSort extends React.Component {
     }
 
     if (syncQueryWithUrl && oldQuery !== newQuery) {
-      this.setState({ locallyChangedSearchTerm:  newQuery });
+      const regExp = /((callNumber|subject|name) [<|>])|"*/ig;
```

Comment:
@UladzislauKutarkin I don't think hard-coding this regex is stripes-smart-components is a good idea since it's intended to be a reusable component. Perhaps what is needed is another prop you pass down from the parent.